image/image.html

```diff
@@ -3,12 +3,54 @@
     category: 'output',
     color: '#a6bbcf',
     defaults: {
+      active: {value:true},
       name: { value: "" }
     },
     inputs: 1,
     outputs: 0,
     icon: "watch.png",
     align: 'right',
+    button: {
+        toggle: "active",
+        onclick: function() {
+            var label = this.name||"debug";
+            var node = this;
+            $.ajax({
```

Comment:
Would it make sense to move these response handlers somewhere outside the `onclick` handler scope?

Reply:
Haven't done that.  Not really sure what you want to achieve with it.  Readability ??